test/index.test.ts

```diff
@@ -229,6 +229,15 @@ for (const { input, expected } of [
       renderer: 'Mali-G51',
     },
   },
+  {
+    expected: {
+      gpu: 'nvidia evga geforce gtx 970',
```

Comment:
This still doesn't feel great but I think it's an issue with the distance between the formatted renderer and the potential matches. I would expect the best match to be `nvidia geforce gtx 970` from the benchmark database but I think the extra cruft in the detected renderer is throwing it off.

@puckey have you considered tokenizing + sorting the strings to handle these weird ordering cases? Here's an [article](https://chairnerd.seatgeek.com/fuzzywuzzy-fuzzy-string-matching-in-python/) from SeatGeek that includes some thoughts on this topic (see the **Token Sort** and **Token Set** sections)

Below is applying this tokenized + de-duping distance logic from their Python library, and you can see that it scores the target I'd expect better.
```python
>>> from thefuzz import fuzz
>>> fuzz.token_set_ratio("nvidia, nvidia nvidia geforce gtx 970 (0x000013c2)), nvidia ", "nvidia evga geforce gtx 970")
90
>>> fuzz.token_set_ratio("nvidia, nvidia nvidia geforce gtx 970 (0x000013c2)), nvidia ", "nvidia geforce gtx 970")
100
```

Reply:
https://github.com/sidsethupathi/detect-gpu/commit/c9e32ba6a9feef546d60573ba47925dffded896b

Here's a pretty rough implementation, but surprisingly only broke 3 tests, one of which was the one I added in this PR. I made the test changes and they seem to be better matches. If this is something you'd like to pursue, happy to clean up the code and submit a PR.

Reply:
Looks great! Would this be IE 11 compatible?

Reply:
The way I currently have it in https://github.com/sidsethupathi/detect-gpu/commit/c9e32ba6a9feef546d60573ba47925dffded896b, possibly not. I think `Array.from` depends on a Set iterator and [it doesn't look like that is supported in IE 11](https://caniuse.com/mdn-javascript_builtins_set_--iterator).

I will tidy up the commit for compatibility and open a new PR. I think this PR (#102) can still be reviewed (and merged) independently if you'd like.